source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,346 @@ Sorts Builders
    :class: singlecol
 
 .. _sorts-builders:
+
+Overview
+--------
+
+In this guide, we show you how to use **builders** to specify
+**sort criteria** for your queries.
+
+Sort criteria are the rules MongoDB uses to sort your data. Some examples of sort criteria are:
+
+* Smallest number to largest number
+* Earliest time of day to latest time of day
+* Alphabetical order by first name 
+
+Builders are classes provided by the MongoDB Java driver that allow you to use
+the builder pattern to interact with your MongoDB instance.
+The builder pattern allows you to create complex
+immutable objects separately from their representation. The
+:doc:`builders </fundamentals/builders/>`
+provided by the MongoDB Java driver can improve the readability of your code.
```

Comment:
Nitpick: use the full 80 column width before wrapping text. Personally, the only exceptions I use are when there's some RST limitation (such as when using "|") or if there is a long link (keep it on a single line).

Reply:
Thanks for the note. Will try to use the entire column width going forward. Fix: https://github.com/mongodb/docs-java/pull/59/commits/5a29fab1d881416b50a9661eeba7913638388498